lib/active_record_host_pool/pool_proxy.rb

```diff
@@ -129,7 +129,16 @@ def _connection_pools
     end
 
     def _pool_key
-      @_pool_key ||= "#{@config[:host]}/#{@config[:port]}/#{@config[:socket]}/#{@config[:username]}/#{@config[:slave] && 'slave'}"
+      @_pool_key ||= begin
+        [
+          @config[:host],
+          @config[:port],
+          @config[:socket],
+          @config[:username],
+          @config[:database],
+          @config[:slave] ? 'slave' : ''
```

Comment:
not sure if there is only true/false in the key, but if there is, this could work too:
```suggestion
          @config[:slave]
```

Reply:
🤔 It looks like like it's always truthy or falsy although `nil` and `false` would produce different values with the `Array#join`. That seems okay though, but it does remove the readability of `slave` in the pool key if you reading the produced pool key at runtime.

I could go either way but lean towards readability. Care to push me over the fence or pull me?